cmd/performance-profile-creator/cmd/root.go

```diff
@@ -465,6 +472,7 @@ func getProfileData(args ProfileCreatorArgs, cluster ClusterData) (*ProfileData,
 	kernelArgs := profilecreator.GetAdditionalKernelArgs(args.DisableHT)
 	profileData := &ProfileData{
 		reservedCPUs:           reservedCPUs.String(),
+		offlinedCPUs:           "", //todo assing value when calculated
```

Comment:
OK, this (and the comment above) are fixed in the next commit. In the future please consider fixing them in the same commit

Reply:
ok